hedera-node/test-clients/src/main/resource/contract/contracts/MultiversionBurn/MultiversionBurn.json

```diff
@@ -0,0 +1,75 @@
+[
```

Comment:
would sort of be nice here (but not required) to use the solidity compiler option to also output method hashes (i.e., selectors) as that would help verifying the various `*_INPUT_V?` constants.

Reply:
I'll do that in the future once I figure out how to do that with hardhat.  I'm not sure how helpful it'll be in this case as the abiEncoding is done within the contract code.